example.go

```diff
@@ -0,0 +1,341 @@
+	for _, assignment := range result.ReplicaAssignments {
+		fmt.Printf("\tassignment:  \t\tid:%d,  \tbrokers: %+v\n", assignment.ID, assignment.Brokers)
+	}
+
+	return nil
+
+}
+
+func createTopic(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Set the retries policy.
+	serviceAPI.EnableRetries(0, 0)
+
+	// Construct an instance of the createTopicOptionsModel.
+	createTopicOptionsModel := new(adminrestv1.CreateTopicOptions)
+	createTopicOptionsModel.Name = core.StringPtr("test-topic")
+	createTopicOptionsModel.Partitions = core.Int64Ptr(int64(26))
+	createTopicOptionsModel.PartitionCount = core.Int64Ptr(int64(1))
+	createTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Create the Topic.
+	response, operationErr := serviceAPI.CreateTopic(createTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Creating Topics: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Creating Topic: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s created\n", *createTopicOptionsModel.Name)
+
+	return nil
+}
+
+func deleteTopic(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Construct an instance of the DeleteTopicOptions model
+	deleteTopicOptionsModel := new(adminrestv1.DeleteTopicOptions)
+	deleteTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	deleteTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Delete Topic
+	response, operationErr := serviceAPI.DeleteTopic(deleteTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Deleting Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Deleting Topic: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s deleted\n", *deleteTopicOptionsModel.TopicName)
+	return nil
+}
+
+func updateDetails(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Construct an instance of the UpdateTopicOptions model
+	updateTopicOptionsModel := new(adminrestv1.UpdateTopicOptions)
+	updateTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	updateTopicOptionsModel.NewTotalPartitionCount = core.Int64Ptr(int64(6))
+	updateTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Invoke operation with valid options model (positive test)
+	response, operationErr := serviceAPI.UpdateTopic(updateTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Updating Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Updating Topics: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s updated\n", *updateTopicOptionsModel.TopicName)
+
+	return nil
+}
+
+//func replaceMirroringTopicSelection(serviceAPI *adminrestv1.AdminrestV1) error {
```

Comment:
The calling code is commented out, so maybe we could uncomment these methods? WDYT?

Reply:
done and used `// nolint` so the final linting does say they are unused.